libs/expo/betterangels/src/lib/screens/AddEditClient/ContactInfo.tsx

```diff
@@ -76,6 +76,7 @@ export default function ContactInfo(props: IContactInfoProps) {
           required
           placeholder="Enter Email"
           label="Email"
+          autoCorrect={false}
```

Comment:
could you please also add `autoCapitalize={false}`? Not in the ticket but I just realized that emails shouldn't be autocapped 🙏